internal/spentcalories/spentCalories.go

```diff
@@ -14,7 +18,25 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
+
 	// ваш код ниже
+	parts := strings.Split(data, ",")
+	if len(parts) != 3 {
+		return 0, "", 0, errors.New("invalid training data")
+	}
+
+	step, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, "", 0, errors.New("invalid training data")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```